simul/platform/runsimul.go

```diff
@@ -99,6 +105,10 @@ func Simulate(serverAddress, simul, monitorAddress string) error {
 			timeout *= 2
 		}
 		childrenWait.Record()
+		log.Lvl2("Broadcasting start")
+		for _, conode := range rootSC.Tree.Roster.List {
+			rootSC.Server.Send(conode, &startSimulation{})
+		}
```

Comment:
As long as we're in it, I'd say it's better to do it, because `Node` is used as a `Init` function. If the init is not finished, it seems less useful than it can be because we're going back to point 0: the users have to code a synchronization mechanism themself.